recipes/django-leaflet/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "django-leaflet" %}
+{% set version = "0.22.0" %}
+{% set sha256 = "2a1f4f31bff2b3a58158372f7910053670483e3e11f8ac365ce75f3e3a4ac11c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ordereddict  # [py<27]
+    - sphinx
+    - sphinx-autobuild
+
+test:
+  imports:
+    - django-leaflet
```

Comment:
It looks like the import test should be `leaflet`

Reply:
I remove test import because import leaflet need some extra env variables.